### PR TITLE
Bump Version

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: frame/bridge/ethereum/backing/src/lib.rs [hunk 1]
```diff
@@ -220,15 +220,6 @@ decl_module! {
 			0
 		}
 
-		fn on_runtime_upgrade() -> frame_support::weights::Weight {
```
Comment (HackFisher) [inline frame/bridge/ethereum/backing/src/lib.rs:223]: Remove this will make darwinia has no effect on runtime upgrade?

**Review by HackFisher — CHANGES_REQUESTED** [review]

File: frame/bridge/ethereum/backing/src/lib.rs (hunk 1 above)
Comment (AurevoirXavier) [reply frame/bridge/ethereum/backing/src/lib.rs:223]: We've 3 runtimes pangolin, crab, darwinia. So I recommend to do a custom runtime upgrade on their runtime.

**Comment by AurevoirXavier** [comment]
@AsceticBear DVM CI failed.

**Review by AsceticBear — APPROVED** [review]

**Review by HackFisher — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
Migration https://github.com/darwinia-network/darwinia/pull/588/commits/d1dc15125f51a751999b400f8d86aa673fb8a92a

**Pull request merged**